app/assets/stylesheets/admin/questions/_questions.scss

```diff
@@ -78,3 +78,12 @@ section.questionnaires li.jqtree_common .jqtree-element
 .linked-document .help-block {
   margin: .25em 0;
 }
+
+.answer-wrapper.tree-view .percentage-input.form-element {
```

Comment:
Not loving how fragile this is; need all these selectors to override simple_form styles.